tests/e2e.test.js

```diff
@@ -110,6 +110,12 @@ describe('External links', () => {
       await expect(page).toClick('a', { title });
     }),
   );
+
+  const footerRepositoryTitle = 'github repository';
+
+  it(`is possible to click the link for Github repository`, async () => {
+    await expect(page).toClick('a', { footerRepositoryTitle });
```

Comment:
This test doesn't seem to work, changing `footerRepositoryTitle` doesn't affect it on my end. Currently, it seems to just tests if there is _any_ link that can be clicked. (and, actually, the same goes for line 110).

You can find the documentation for [`toClick` here](https://github.com/smooth-code/jest-puppeteer/blob/master/packages/expect-puppeteer/README.md#expectinstancetoclickselector-options). I assumed, `{ title }` would look at the `title` attribute, but it seems that's not a thing that's supported.

While we're talking about this, it's not quite clear to me why you define `footerRepositoryTitle` outside the scope of the test and why you use backticks for the test name.

In this case you can fix the test using, e.g.:

```suggestion
    await expect(page).toClick('a', { text: /Made with . on GitHub/ });
```

Reply:
fixed.